app/assets/stylesheets/reskin/moneyhelper_styling_common.scss

```diff
@@ -0,0 +1,210 @@
+// -- Headers ----------------------------------------------------------------------
+h1,h2,h3,h4,h5,h6{
+    font-family: 'Roobert-Bold', 'Times';
+    font-style: normal;
+    font-weight: bold;
+    color: $blue_primary;
+}
+
+// ---------------------------------------------------------------------------------
+
+
+// -- Paragraphs -------------------------------------------------------------------
+p{
+    font-family: 'Roobert-Regular', sans-serif!important;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 23px;
+    color: $text_secondary;
+    &.callout{
+        background-color: $light_grey;
+        border-radius: $border_radius;
+        padding: 10px;
+    }
+}
+// ---------------------------------------------------------------------------------
+
+// -- Links ------------------------------------------------------------------------
+    a:not(.navigation__link){
+        color: $magenta_primary!important;
+        text-decoration: underline;
+        &:hover{
+            color: $magenta_dark;
+            text-decoration: none;
+        }
+        &:active{
+            color: $magenta_darker;
+            text-decoration: none;
+        }
+        &:focus{
+            color: $text_secondary;
+            background-color: $yellow_secondary;
+            border-bottom: 4px solid $support_focus;
+            text-decoration: none!important;
+            outline: none;
+        }
+    }
+// ---------------------------------------------------------------------------------
+
+// -- Buttons ----------------------------------------------------------------------
+.button{
+    box-sizing: border-box;
```

Comment:
This is defined in the existing styles and will be inherited so it not needed here.